cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Oh right that is because the file that I commited was a work in progress that got added to the PR, accidentally.

Sorry for the confusion caused. I will push the new files that actually have the xml inclusion.